ios/FileUtils.m

```diff
@@ -108,36 +163,69 @@ @implementation FileUtils
  * @returns The height and width (x,y), of the video or image in pixels.
  */
 RCT_EXPORT_METHOD(
-                  getDimensions:(NSString *)path (NSString *)type
+                  getDimensions:(NSString *)path
+                  type: (NSString *)type
                   resolver: (RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject
                   )
 {
-    if (type == "image") {
-        UIImage *image = [UIImage imageWithContentsOfFile:path];
+    NSURL *referenceUrl = [NSURL URLWithString:path];
+    
+    if ([type isEqualToString:@"image"]) {
+        CGImageSourceRef sourceRef = CGImageSourceCreateWithURL((CFURLRef)referenceUrl, NULL);
         
-        if (image == nil) {
+        if (sourceRef == nil) {
             reject(
-                   @"QSRNFU-20",
+                   @"QSRNFU-30",
```

Comment:
Checkout the Expo error format: https://github.com/expo/expo/pull/15615/files#diff-e916d516bcb79a8e6a9b51fe60b514130b1c20c48620bcff4480e91253af734dL648 

```
reject(@"ERR_FILE_NOT_EXISTS", ..., nil);
``` 

Originally I'd use the same format as you above for error tags, although I think Expo does it better as it's more descriptive. What dp you think?

Reply:
Oh yeah, totally agree. I saw it in a couple libraries, so I though it was a standard for iOS... but happy to replace those.